pkgs/games/warzone2100/default.nix

```diff
@@ -53,6 +54,16 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-AdYI9vljjhTXyFffQK0znBv8IHoF2q/nFXrYZSo0BcM=";
   };
 
+  patches = [
+    # Upstream patch for curl-8.4 support,
+    # TODO: remove on next release.
+    (fetchpatch {
+      name = "curl-8.4.patch";
+      url = "https://github.com/Warzone2100/warzone2100/commit/df072b0d910ab7ded37b737e8dcc297bfccf6f6b.patch";
```

Comment:
> Upstream patch

> This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository.

Where did you find the patch?

Reply:
Found it https://github.com/Warzone2100/warzone2100/commit/db1cf70950d4fa6630f37a7bf85f548b48ed53cd

Please update the revision

Reply:
Updated.

I think I navigated to the specific commit via one of the pull requests and then added a `.patch` to it. Probably pull request just expired. I keep being bitten by it when upstream does squash/rebase. My apologies.